src/fuzzer/seed_corpus/SecurityBaselineMmiGet.auditEnsureAcceptingSourceRoutedPacketsIsDisabled.target

```diff
@@ -0,0 +1 @@
+SecurityBaselineMmiGet.auditEnsureAcceptingSourceRoutedPacketsIsDisabled
```

Comment:
>ecurityBaselineMmiGet.auditEnsureAcceptingSourceRoutedPacketsIsDisabled [](http://example.com/codeflow?start=1&length=71)

There is no need to fuzz at this level, per individual audit* and remediate* call in asb lib. It is 100% the same to audit the exports from asb lib, the AsbMmiGet and AsbMmiSet, which directly translate to these. 

Both have same pros and cons, byt doing it this way complicates things for no reason.

Please consider:

1. Fuzz at commonutils, not asb, level.
2. If at asb lib level, fuzz the top exports, not the individual rules.

Reply:
As discussed, the list of targets is going to change in future, I am going to go over your email suggestions and add them up accordingly. Can you clarify what do you mean by 'top exports'?

Reply:
By top exports for asb library mean AsbIsValidResourceIdRuleId and AsbMmiGet. Originally in my comment I stated, quote: 'exports from asb lib, the AsbMmiGet and AsbMmiSet' but AsbMmiSet is too dangerous to fuzz inputs on. Think we all agreed on the e-mail exchange on this and now this conversation can be closed.